pyodide-build/pyodide_build/cli/build.py

```diff
@@ -130,6 +134,10 @@ def main(
     build_dependencies: bool = typer.Option(
         False, help="Fetch non-pyodide dependencies from pypi and build them too."
     ),
+    output_lockfile: bool = typer.Option(
+        False,
+        help="Output list of resolved dependencies alongside the build in dist/package_versions.txt",
```

Comment:
As @rth mentioned, I think it would be nice to make this flag to specify a path of a lockfile, instead of a boolean flag. So that we are a bit less opinionated about the file name of this lock file or its location.